.github/workflows/build-sonarqube.yml

```diff
@@ -0,0 +1,55 @@
+name: "[Bot] Run sonarqube code scanning'"
+run-name: "@Desktop • Test App triggered by ${{ github.event_name == 'workflow_dispatch' && inputs.login || github.actor }} ${{ format('on ref {0}', github.ref_name) }}"
+
+on:
+  push:
+    branches:
+      - develop
+      - support/LIVE-9378-add-sonarqube-integration
+  workflow_dispatch:
```

Comment:
Why we would run it manually ?

Reply:
@mbertin-ledger nice catch, this is not needed. I copied the desktop testing workflow for the initial setup, the Sonarqube testing step will possibly be moved to test.yml flow after @gre's feedback so we won't run the app tests various times.